### DISCUSSION TITLE
Newcomer Contributions

### DISCUSSION THREAD

**Discussion by MNMelton** [mlflow/mlflow#12949] [General]
Hey everyone!

I'd love to explore some of the feature requests, but the good first issue tag seems a pinch outdated, with people wanting to volunteer but no one with experience in the code base signing off on who can work on it. I want to avoid stepping on anyone's toes since some people have voiced their desire to work on those issues. With that being said, would it still be best to jump into those issues or would you rather have newcomers dive into other issues that don't have much support?

**Comment by B-Step62** [comment]
@MNMelton Thank you for your willingness to contribute! Yes that's fine, we don't use strict 'assign-and-work' model. Of course there should be some moral - you should not cut in aggressively just a day after someone else volunteers. However, generally it is welcomed to come forward if the issue doesn't have any progress for a while - say 1 week. I would also recommend tagging the original volunteer and wait for their response for a day or two before actually taking over. Looking forward to your first contribution!!🙂
